Sources/Client/Model/Filter.swift

```diff
@@ -210,3 +212,58 @@ public extension Filter {
         lhs = lhs | rhs
     }
 }
+
+// MARK: - Key Helper
+
+public extension Filter.Key {
+    
+    /// A filter with a key of the string equal to a given value.
+    func equal(to value: Encodable) -> Filter {
```

Comment:
I think it's problematic to have a function with signature `equal(to: Encodable)` other developers might already have this. Same for others (`notEqual`, `greater`, etc..)
I suggest adding `filter` prefix to them? Or `createFilterWith` (`createFilterWithEqual`, `createFilterWithGreater`). None of the suggestions look as elegant as the current structure, but I wanted to point out possible name clashes.

Reply:
The signature has a specific parameter type and return type. I don't think this could be a problem. If we got a request for that, we will change it.